src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -532,21 +537,25 @@ private void Poll()
         private List<ConsumeResult<K, V>> PollKafka(CancellationToken token)
         {
             ConsumeResult<K, V> consumed = null;
-            var timeout = (int) _pollTimeout.TotalMilliseconds;
+            var i = 10; // 10 poll attempts
```

Comment:
Previous code only ran long enough for a single poll attempt. Break apart the timeout into 10 and try it.

Reply:
Note: need to test this without a `PollTimeout` set